docs/source/searching.rst

```diff
@@ -99,3 +99,20 @@ The metadata from those columns then will be included in the results.
 In order to specify multiple columns, simply repeat the query string parameter once for every column name (the ``requests`` module will do this automatically for you if you pass it a list of values for a query string parameter).
 
 In order to retrieve all metadata columns that the catalog knows about, use ``metadata_fields=_all``.
+
+
+Retrieving full objects
+-----------------------
+
+If the data provided as metadata is not enough, you can retrieve search results as full serialized objects equivalent to what the resource GET request would produce.
+
+You do so by specifying the ``fullobjects`` parameter:
+
+.. code-block:: http
+
+    GET /plone/@search?fullobjects&SearchableText=lorem HTTP/1.1
+    Accept: application/json
+
+.. warning::
+
+    Be aware it might induces performance issues when retrieving a lot of resources. Regular search just uses lazy objects, but with full objects retrieving we wake up all the returned objects.
```

Comment:
=> `Be aware that this might induce performance issues when retrieving a lot of resources. Normally the search just serializes catalog brains, but with full objects we wake up all the returned objects.`

(*Resultsets are still lazy, in the sense that only the objects in the returned batch(es) are woken up*)